src/gateway/ProjectStateManager.java

```diff
@@ -69,4 +70,9 @@ public int getNumTeams() {
     public void saveTeam(Team t) {
         teams.put(t.teamName, t);
     }
+
+    @Override
+    public MemberTask getMemberTask(String e){
+        return null;
```

Comment:
```suggestion
        return memberTasks.get(e);
```

Reply:
Need to add at top...

protected ConcurrentHashMap<String, MemberTask> memberTasks;